### PR TITLE
feat: support ioredis v5

### LINKED ISSUE #1164: ioredis v5 filename case change breaks import of built/command.ts
I have upgraded ioredis to v5 and noticed that my tests work fine on Mac but not on the CI running Linux. 

```
 Cannot find module 'ioredis/built/command' from '../../node_modules/ioredis-mock/lib/index.js'
```

As it turns out, the casing for the module changed in ioredis v5 from **built/command.ts** to **built/Command.ts**. OSX is case-insensitive, so tests were still working locally but not when I ran them trough the linux-based deployment pipeline.

additional note, just saw this pull request which I believe fails for the described reason: [1163](https://github.com/stipsan/ioredis-mock/pull/1163)

Greets
-act

### PR DESCRIPTION
**Pull request by luin**
Closes #1164

I don't have Linux so can't test in a case-sensitive environment 🤣.

Test plan:

1. Make sure tests pass with ioredis v4 and ioredis v5.

### CODE REVIEW

**Comment by zoltan-nz** [comment]
Looks good to me, hope we can merge it soon. Thanks for providing this update.

File: src/command.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
-import IoredisCommand from 'ioredis/built/command'
 import asCallback from 'standard-as-callback'
 
 import promiseContainer from './promise-container'
 
+let IoredisCommand
+try {
+  // ioredis v4
+  // eslint-disable-next-line global-require, import/no-unresolved
+  IoredisCommand = require('ioredis/built/command').default
```
Comment (vamsiampolu) [inline src/command.js:9]: the require statement needs to be `ioredis/built/Command`.

**Comment by crobinson42** [comment]
#1172

**Comment by danieldspx** [comment]
@luin I came up with a simillar solution (#1176) because I didn't see your PR. I can confirm that the tests would pass. Except for the fact that the `esbuild` will try to bundle `ioredis/built/Command` and `ioredis/built/command` but it won't be able because we do not have both versions installed. Therefore, we should set them as a external depedency by addind `--external:ioredis/built/Command --external:ioredis/built/command` to the `build:browser.js` script.

**Comment by danieldspx** [comment]
@stipsan could you please merge and publish a new minor version with this feature? It does not make sense to downgrade my ioredis just to be able to use ioredis-mock. Otherwise I will publish it myself under another name (I do not wish to do that but I really need this feature :smile:).

**Comment by stipsan** [comment]
On it 👍

**Comment by stipsan** [comment]
@danieldspx the esbuild problem can be solved by changing the ordering to test `ioredis` v5 first, and set that to the `devDependency` then set the v4 path as an external 😄  will fix!

**Pull request merged**